tests/v2/validation/Jenkinsfile

```diff
@@ -102,6 +102,11 @@ node {
                   }
                 }
 
+                dir("./tests/v2/validation/.ssh") {
+                  def decodedRsa = new String(AWS_SSH_RSA_KEY.decodeBase64())
+                  writeFile file: JENKINS_RKE_VALIDATION, text: decodedRsa
+                }
```

Comment:
just checking, this works if there's no `AWS_SSH_RSA_KEY` defined too? or are we safe to assume it will always be there?

Reply:
This is already preconfigured in our Jenkins environment, so this should always be fine to assume. And if it were ever deleted, we could just add back in pretty easily.